src/main/java/customLambdaFunctions/NullChecker.java

```diff
@@ -0,0 +1,5 @@
+package customLambdaFunctions;
+
+public interface NullChecker<T> {
```

Comment:
```suggestion
public interface INullChecker<T> {
```
use same naming conventions for functional interfaces